### PR TITLE
RONDB-203: Fix mysqld complaint "mbind: Operation not permitted"

### PR DESCRIPTION
**Pull request by svenssonaxel**

### CODE REVIEW

File: build_run_docker.sh [hunk 1]
```diff
@@ -469,6 +469,11 @@ if [ $NUM_MYSQL_NODES -gt 0 ]; then
         template+="$command"
         # template+="$RESOURCES_SNIPPET"
 
+        # mysqld needs this, or will otherwise complain "mbind: Operation not permitted".
```
Comment (olapiv) [inline build_run_docker.sh:497]: Do you mean when you use valgrind? Since I've never seen that error message before.

Comment (svenssonaxel) [reply build_run_docker.sh:497]: Hmm, not sure actually. I'll check that, and if so I'll include this in the (future) instrumentation PR instead.

Comment (olapiv) [reply build_run_docker.sh:497]: Okay cool 👍🏼

Comment (svenssonaxel) [reply build_run_docker.sh:497]: I can confirm I'm getting this without valgrind, but it seems I need to actually start benchmarks first.

**Review by olapiv — APPROVED** [review]

**Pull request merged**